### ISSUE TITLE
Quantization of MaxVit model

### ISSUE THREAD

**Issue by mkompanek** [intel/neural-compressor#1576]
Hi, I am trying to quantize torchvision's MaxVit (disregarding accuracy for the sake of simplicity). Resulting model always ends up slower than FP32 version. Is this architecture not supported, is there anything I can do better ?

```python
import neural_compressor
from neural_compressor import quantization
import numpy as np
import onnx
import onnxruntime
import time
import torch
import torchvision

def benchmark(model_path: str, runs: int = 10) -> None:
    session = onnxruntime.InferenceSession(model_path)
    total = 0.0    
    input_data = np.random.randn(1, 3, 224, 224).astype(np.float32)
    _ = session.run([], {'input': input_data})
    for i in range(runs):
        start = time.perf_counter()
        _ = session.run([], {'input': input_data})
        total += (time.perf_counter() - start) * 1000
    print(f"Avg: {total/runs:.2f}ms")

class FakeLoader:
    def __init__(self) -> None:
        self.batch_size=1 
    def __iter__(self):
        for _ in range(10):
            yield np.random.randn(1, 3, 224, 224).astype(np.float32), 0

if __name__ == '__main__':
    onnx_path = 'torch_model.onnx'
    quantized_path = 'torch_model_quantized.onnx'

    torch_model = torchvision.models.maxvit_t(weights=torchvision.models.MaxVit_T_Weights.IMAGENET1K_V1).eval()

    torch.onnx.export(torch_model, torch.rand((1, 3, 224, 224)),
                      onnx_path, input_names=['input'], output_names=['output'], opset_version=14) 

    onnx_model = onnx.load(onnx_path)
    config = neural_compressor.PostTrainingQuantConfig(domain='cv', approach='static', device='cpu')

    q_model = quantization.fit(onnx_model, config, calib_dataloader=FakeLoader())
    q_model.save(quantized_path)

    benchmark(onnx_path)
    benchmark(quantized_path)```

**Comment by yiliu30** [comment]
Hi @mkompanek, thanks for raising this issue. I replicated it on my end. It appears that the quantized model has a large amount of `QuantizeLinear` ops and `DequantizeLinear` ops. 

Log:
```bash
2024-01-27 22:46:51 [INFO] Quantize the model with default config.
2024-01-27 22:47:16 [INFO] |********Mixed Precision Statistics*******|
2024-01-27 22:47:16 [INFO] +-------------------+-------+------+------+
2024-01-27 22:47:16 [INFO] |      Op Type      | Total | INT8 | FP32 |
2024-01-27 22:47:16 [INFO] +-------------------+-------+------+------+
2024-01-27 22:47:16 [INFO] |        Conv       |   61  |  61  |  0   |
2024-01-27 22:47:16 [INFO] |       MatMul      |   90  |  90  |  0   |
2024-01-27 22:47:16 [INFO] |        Mul        |  179  |  44  | 135  |
2024-01-27 22:47:16 [INFO] |      Sigmoid      |   22  |  22  |  0   |
2024-01-27 22:47:16 [INFO] | GlobalAveragePool |   12  |  12  |  0   |
2024-01-27 22:47:16 [INFO] |        Pad        |   4   |  4   |  0   |
2024-01-27 22:47:16 [INFO] |        Add        |  301  | 144  | 157  |
2024-01-27 22:47:16 [INFO] |      Reshape      |  176  | 154  |  22  |
2024-01-27 22:47:16 [INFO] |    AveragePool    |   4   |  4   |  0   |
2024-01-27 22:47:16 [INFO] |     Transpose     |  165  | 132  |  33  |
2024-01-27 22:47:16 [INFO] |      Flatten      |   1   |  1   |  0   |
2024-01-27 22:47:16 [INFO] |       Slice       |   66  |  66  |  0   |
2024-01-27 22:47:16 [INFO] |   QuantizeLinear  |  125  | 125  |  0   |    <-------------- quantization overhead
2024-01-27 22:47:16 [INFO] |  DequantizeLinear |  169  | 169  |  0   |    <-------------- quantization overhead
2024-01-27 22:47:16 [INFO] +-------------------+-------+------+------+
2024-01-27 22:47:16 [INFO] Pass quantize model elapsed time: 25353.2 ms
```

The issue arises because the model structure is quite complex and the `QuantizeLinear` and `DequantizeLinear` ops aren't fused,
which results in extra overhead. If you want to know more about the kernel calls, you can utilize the [profiler script](https://github.com/microsoft/onnxruntime/blob/6d7ac9c93ae7cd1f979b35e7f6d7af207962cd99/onnxruntime/python/tools/transformers/profiler.py) to get model details.

**Comment by yiliu30** [comment]
Since there hasn't been a response for a while, close it first. Feel free to reopen it if you need further assistance. Thank you!
